bin/cron/upload_to_mapbox.rb

```diff
@@ -0,0 +1,45 @@
+# Util file for uploading tilesets to Mapbox
+# Tilesets should be generated using Tippecanoe.
+# Tilesets are uploaded to Mapbox's servers using the process documented at
+# https://docs.mapbox.com/api/maps/#uploads.
+#
+# Before using these functions, be sure to provide the following CDO attribute:
+#  CDO.mapbox_upload_token: (the auth token for uploading tiles to Mapbox's servers)
+
+require 'aws-sdk-s3'
+require 'rest-client'
+
+def create_upload_credentials
+  puts "Creating credentials"
+  response = RestClient.post("https://api.mapbox.com/uploads/v1/bethanycodeorg/credentials?access_token=#{CDO.mapbox_upload_token}", {})
```

Comment:
Can we get an org-wide account/URL that doesn't have your name in it? :)

Reply:
Just set one up! I'll update this PR to use that account.

Reply:
`codeorg` account is setup with credentials in the Shared Engineering folder in LastPass :)